package.json

```diff
@@ -1,36 +1,34 @@
 {
-    "name": "to-do-list",
-    "version": "1.0.0",
-    "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
-    "private": true,
-    "scripts": {
-      "test": "echo \"Error: no test specified\" && exit 1",
-      "build": "webpack"
-    },
-    "repository": {
-      "type": "git",
-      "url": "git+https://github.com/david-lafontant/To-Do-List.git"
-    },
-    "keywords": [],
-    "author": "",
-    "license": "ISC",
-    "bugs": {
-      "url": "https://github.com/david-lafontant/To-Do-List/issues"
-    },
-    "homepage": "https://github.com/david-lafontant/To-Do-List#readme",
-    "devDependencies": {
-      "babel-eslint": "^10.1.0",
-      "css-loader": "^6.5.1",
-      "eslint": "^7.32.0",
-      "eslint-config-airbnb-base": "^14.2.1",
-      "eslint-plugin-import": "^2.25.4",
-      "html-webpack-plugin": "^5.5.0",
-      "style-loader": "^3.3.1",
-      "webpack": "^5.66.0",
-      "webpack-cli": "^4.9.1"
-    },
-    "dependencies": {
-      "lodash": "^4.17.21"
-    }
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
+  "private": true,
+  "scripts": {
+    "start": "webpack --config webpack.config.js --watch",
+    "build": "webpack serve --open --config webpack.config.js"
+  },
```

Comment:
- [ ] Good job implementing this change, however, on `npm run build` the project still does not build or opens in the browser window unless I choose to manually install the `webpack-dev-server` package.

![image](https://user-images.githubusercontent.com/57421823/151050449-902be8e6-59e0-4611-8ad0-c22935b40759.png)

Kindly add to your dependency `webpack-dev-server` as a dev dependency: `npm install -D webpack-dev-server` and ensure that the project builds and starts with the commands (it should work with the dependency installed in your package.json).

If on doubt, check [this](https://webpack.js.org/configuration/dev-server/)

That way, you will be working directly with webpack on the terminal and your project will auto-update the live preview! 💪

Reply:
I added the package as required. Given by the precedent reviewer `"start": "webpack --config webpack.config.js --watch"`,`npm start` starts the server, but doesn't run the browser automatically. However with the start parameter defined like this `"start": "npm run build"`, the server opens and the page is displayed in the default browser automatically. 
You asked that the project works with the commands(???), if the command is `npm start`, I think I have solved this issue. However, in order to do that, I changed the parameter, hoping it's not something sacred. 
Best regards.

Reply:
### Hey @david-lafontant,  the only issue that was preventing the browser preview was that `webpack-dev-server` was not installed in package.json:

![image](https://user-images.githubusercontent.com/57421823/151072782-7bf2ef27-3139-4c3d-a203-0b382debac0b.png)

### Upon cloning the project and trying to run it for the first time, this prompt appeared, asking for the installation of the missing dev dependency:


![image](https://user-images.githubusercontent.com/57421823/151050449-902be8e6-59e0-4611-8ad0-c22935b40759.png)

### And that is why the last reviewer required this:

![image](https://user-images.githubusercontent.com/57421823/151074025-67528637-3565-40ef-a80d-a910171c694e.png)



### I can see that you already included in recent commits the dependency on package.json so don't worry it should work now 👍

Reply:
I'll share with you my personal package.json for webpack projects, I hope it helps:

![image](https://user-images.githubusercontent.com/57421823/151074586-1140ead7-30ff-4068-a91c-e1009e80e479.png)

Happy coding 💪

Reply:
> I'll share with you my personal package.json for webpack projects, I hope it helps:
> 
> ![image](https://user-images.githubusercontent.com/57421823/151074586-1140ead7-30ff-4068-a91c-e1009e80e479.png)
> 
> Happy coding 💪

Thank you very much.